src/edit/InstanceForm.js

```diff
@@ -342,7 +342,7 @@ class InstanceForm extends React.Component {
     ) : [];
 
     const statisticalCodeOptions = referenceTables.statisticalCodes
-      .map(
+      ?.map(
```

Comment:
Ah, now I see what happened above: here, there was no ternary and you solved the bug with optional chaining, and so added optional chaining everywhere. It doesn't matter which we choose, but let's pick one pattern and stick with it.